apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import Image from "next/image";
+
+import Accordion from "react-bootstrap/Accordion";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+import clip from "@/assets/images/clip.svg";
+
+import styles from "./ClipboardSchedule.module.scss";
+
+function ClipboardSchedule() {
+	return (
+		<Container
+			as="section"
+			className={styles.clipboard + " px-0 position-relative"}
+		>
+			<div className={styles.clip}>
+				<Image src={clip} alt="Clipboard clip" className={styles.clip} />
+			</div>
+			<h2 className="mb-5">Countdown Timer</h2>
+			<Accordion defaultActiveKey="0">
+				<Accordion.Item eventKey="0" className={styles.accordionItem}>
+					<Accordion.Header as="div" className={styles.accordionHeader}>
+						<Row className="w-100 justify-content-between align-items-center">
+							<Col sm="auto">
+								<h3 className="text-start">Event Name</h3>
+							</Col>
+							<Col sm="auto" className="me-3">
+								<span>DBH 4011, 1-2pm</span>
+							</Col>
+						</Row>
+					</Accordion.Header>
+					<Accordion.Body>Placeholder text</Accordion.Body>
+				</Accordion.Item>
```

Comment:
Chore: Even with placeholder data, we still eventually need to parameterize a component for the individual accordion items.

Reply:
Turns out there needs to be some fixes with the event schema. I converted the `AccordionItem`s to a loop over a list of events that will be passed in through props.